build_scripts/build_windows-2-installer.ps1

```diff
@@ -4,7 +4,7 @@ $ErrorActionPreference = "Stop"
 
 mkdir build_scripts\win_build
 
-git status
+git submodule
```

Comment:
Thinking this was maybe an unintentional change?

Reply:
Adding `git submodule` was intentional, but deleting `git status` was unintentional...
I'll fix this.